.circleci/config.yml

```diff
@@ -512,6 +512,52 @@ jobs:
           name: Upload coverage
           command: codecov --verbose --clean --flags <<parameters.coverage_flag>>
 
+  sdk-next-tests:
+    docker:
+      - image: ethereumoptimism/ci-builder:latest
+    resource_class: large
+    steps:
+      - checkout
+      - attach_workspace: { at: '.' }
+      - check-changed:
+          patterns: sdk,contracts-bedrock,contracts
+      - restore_cache:
+          name: Restore Yarn Package Cache
+          keys:
+            - yarn-packages-v2-{{ checksum "yarn.lock" }}
+      - run:
+          name: anvil-l1
+          background: true
+          command: anvil --fork-url $ANVIL_L1_FORK_URL
+      - run:
+          name: anvil-l2
+          background: true
+          command: anvil --fork-url $ANVIL_L2_FORK_URL --port 9545
+      - run:
+          name: build
+          command: yarn build
+          working_directory: packages/atst
+      - run:
+          name: lint
+          command: yarn lint:check
+          working_directory: packages/atst
+      - run:
+          name: make sure anvil l1 is up
+          command: cast block-number --rpc-url http://localhost:8545
```

Comment:
this will need to fork a specific block after this test starts working to make it able to claim the withdrawal on every test run